oap-server/server-core/src/main/java/org/apache/skywalking/oap/server/core/alarm/AlarmRecord.java

```diff
@@ -64,6 +66,8 @@ public String id() {
     private long startTime;
     @Column(columnName = ALARM_MESSAGE, matchQuery = true)
     private String alarmMessage;
+    @Column(columnName = LABELS, matchQuery = true)
+    private String labels;
```

Comment:
Adding this? Are you going to modify UI?

Reply:
I'm not going to modify UI, just store data.
I've just read the code about `@ Column` in other persistent scenario.

```java
@Column(columnName = LABELS, dataType = Column.ValueDataType.LABELED_VALUE)
private DataTable labels;
```

Is that more appropriate? But I have a question, how to search by the specified label(key=value)？